airflow/www/views.py

```diff
@@ -2283,9 +2260,10 @@ def tree(self, session=None):
             .limit(num_runs)
             .all()
         )
-        dag_runs = {dr.execution_date: alchemy_to_dict(dr) for dr in dag_runs}
+        encoded_runs = [wwwutils.encode_dag_run(dr) for dr in dag_runs]
+        dag_run_dates = {dr.execution_date: alchemy_to_dict(dr) for dr in dag_runs}
 
-        max_date = max(dag_runs.keys(), default=None)
+        max_date = max(dag_run_dates.keys(), default=None)
```

Comment:
```suggestion
        max_date = max(dag_run_dates, default=None)
```

I think that `max(dict)` will return the max key by default.